infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -241,7 +227,8 @@ public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
             if (conflictParamSet != null && conflictParamSet.size() > 0) {
                 // for each key pair, check for conflict;
                 // if not present, check against the wildcard
-                conflictParamSet.forEach(conflictParam -> {
+                for (byte[] conflictParam : conflictParamSet) {
```

Comment:
There was an old PR (I think the "precise conflicts" PR) which factored this out.  But you could just return here, yes?